src/main/kotlin/cacophony/parser/CacophonyGrammarSymbol.kt

```diff
@@ -1,70 +1,97 @@
+@file:Suppress("ktlint:standard:no-wildcard-imports")
```

Comment:
This annotation will be unnecessary after merging, since we have enabled wildcard imports in the whole project.